src/core/appinterface.h

```diff
@@ -38,6 +38,8 @@ class AppInterface : public QObject
       Q_ASSERT( false );
     }
 
+    Q_INVOKABLE void importUrl( const QString &url );
```

Comment:
Please add some docs :-)

Reply:
Done, added coverage for other doc-less signals too.